.github/workflows/release.yml

```diff
@@ -437,13 +437,53 @@ jobs:
         path: |
           signed
 
+# ================================
+#           Validate
+# ================================
+  validate:
+    name: Validate installers
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+    needs: [ osx-sign, win-sign, linux-sign ]
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+
+      - name: Build & Install Windows
+        if: contains(matrix.os, 'windows')
+        shell: sh
```

Comment:
Oh, that's allowed? I thought the permissible value was called `bash`...

Reply:
It does seem to be allowed, but I can update it to bash.